__tests__/pages/telekonseling.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import TelecounselingPage, { getStaticProps } from "~/pages/telekonseling";
+import telekonseling from "~/lib/content/telekonseling";
+
+jest.mock("~/lib/content/telekonseling");
```

Comment:
Where did you mock this file? I didn't see any mock file defined in this PR.

Reply:
I just did, hehe.

Honestly, I didn't knew if this mock file exist (and what is it for). When I made ```telekonseling```, I compared ```isolasi-mandiri``` and ```edukasi``` then put the same code from both files to ```telekonseling```. I tracked every code by doing ```CTRL + Left Click``` to make sure where those file connected. Since the mock file returns from ```index```, I missed it.

Reply:
For the icon, yes, I think there's no other way of testing it using the test id.
But we can still improve the query for the title, as I just did in 0ce6e4ea7673c5df17d166df6156ebde4b1f40ec.